mysql-test/suite/galera/t/galera_as_master_gtid.cnf

```diff
@@ -4,3 +4,12 @@
 log-bin=mysqld-bin
 log-slave-updates
 binlog-format=ROW
+[mysqld.1]
+gtid-domain-id=1
+wsrep_gtid_mode=1
+wsrep_gtid_domain_id=100
```

Comment:
Why is there a different `wsrep_gtid_domain_id` to `gtid-domain-id`? Is there a use case? I can't see a use case that would cause these to be different and a few cases that would cause conflicts in mixed async/galera deployment.

Reply:
wsrep_gtid_domain_id is used for events that are replicated in cluster. For example if you have table that uses any other engine than InnoDB, this GTID will not contain wsrep_gtid_domain_id, it will use gtid_domain_id.
Let's say that we don't have this distinction and use only gtid_domain_id. It could happen that on some node GTID seqno is different than on other nodes in cluster (maybe some query that uses table that is not InnoDB SE is used and this event ends up in binlog).
Patch tries to fix that problem, and it should provide consistency of replicated events GTID in cluster.

Reply:
So a non-innodb commit will contain wsrep_gtid_domain_id in the galera replication however gtid_domain_id in the binlog?

I partially follow this rational but you highlight I don't know why non-innodb gtids are currently different.

Reply:
To keep it simple, non-innodb events will not be replicated.